tests/basic_sparse_test.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# This test checks that Qdrant answers to all API mentioned in README.md as expected
+
+set -ex
+
+QDRANT_HOST=${QDRANT_HOST:-'localhost:6333'}
+COLLECTION_NAME=${COLLECTION_NAME:-'sparse_collection'}
+
+# cleanup collection if it exists
+curl -X DELETE "http://$QDRANT_HOST/collections/$COLLECTION_NAME" \
+  -H 'Content-Type: application/json' \
+  --fail -s \
+  --fail -s | jq
+
+# create collection
+curl -X PUT "http://$QDRANT_HOST/collections/$COLLECTION_NAME" \
+  -H 'Content-Type: application/json' \
+  --fail -s \
+  --data-raw '{
+      "vectors": { },
```

Comment:
have we talked about making `vectors` optional already?

Reply:
TBH I have forgotten. I guess it's a save change

Reply:
I propose to do it in a separate PR. It will produce a lot of additional changes

Reply:
Also with optional `vectors` we may discuss validation of configs, dense and sparse vectors must have unique vector names